### DISCUSSION TITLE
媒體識讀，從現在做起

### DISCUSSION THREAD

**Discussion by TW-Maverick** [1101-GS4719-C/Python-Community-GS4719-C#1320] [General]
上禮拜看到錫蘭抱怨台灣的媒體亂象，對確實這已經探討過很多年了，但都不見其改善，就像台灣交通一樣XD。
我直到看完影片才了解到事情的嚴重性，我爛哈哈!

根據2022年牛津大學的研究指出: 依據一般民眾主觀認為
可信度排名 : 1.天下雜誌  2. 商業週刊  3.經濟日報  4.公視  5.TVBS
使用率排名(不包含網路上) : 1. TVBS  2. 東森電視  3. 三立   4.民視

除了TVBS之外，其他使用率很高的媒體其可信度都排在7名之外，這就很可笑了，我們愛看又愛嫌。

這邊分享影片介紹的3個簡單步驟讓你看懂媒體風向
1. 說了什麼? 沒說什麼?
2. 服務了誰? 傷害了誰?
3. 事實優先 or 立場搶先?

通常正常的媒體，事實會先於立場!!!

我覺得要有媒體識讀的能力，有一個重點就是多看不同觀點媒體報導的。
那我就介紹的這邊，希望各位也能看懂媒體風向!!!

🔗參考文章連結
[Reuters Institute Digital News Report 2022](https://reutersinstitute.politics.ox.ac.uk/sites/default/files/2022-06/Digital_News-Report_2022.pdf)

🔗參考影片連結
[住了6個月後，這些是我對台灣真實的看法](https://www.youtube.com/watch?v=-JNeBKlG0cI&ab_channel=%E5%85%AC%E8%A6%96P%23%E6%96%B0%E8%81%9E%E5%AF%A6%E9%A9%97%E5%AE%A4)
[台灣媒體是怎麼爛掉的？](https://www.youtube.com/watch?v=-JNeBKlG0cI&ab_channel=%E5%85%AC%E8%A6%96P%23%E6%96%B0%E8%81%9E%E5%AF%A6%E9%A9%97%E5%AE%A4)

**Comment by KozakHou** [comment]
媒體給啥大家吃啥，總白癡化已經出現了。

就我感覺起來，社會大幅度撕裂了。
社會上中立的人已經屈指可數或是要被逼著選邊站。
媒體識讀的提倡是能減少上述的發生，但我國是全世界假新聞充斥最多的國家，所以感覺實施是相當困難的呀!

**Reply by TW-Maverick** [reply]
你說的我可以認同，但舊的不去新的不來，
先從個人開始培養，
等未來十幾年後，大部分的人就會有媒體識讀了，XD
If 世界還沒滅亡

**Reply by KozakHou** [reply]
當人們只願意相信自己願意相信的時候，世界就變了調、孩子們哭鬧、我很想大叫、跟這世界說不要。

**Comment by bestbanana295** [comment]
是真的，如果想要得到比較靠近真實資訊就不能把眼光只專注在一個媒體上，因為媒體通常都有自己的政治立場從而影響他們所產出的新聞。
我認為，台灣現在的假新聞層出不窮，如果連媒體都有時候沒有查證就把新聞放上來，這也不免造成社會大眾混淆。

**Comment by Jamesmoneya** [comment]
個人看到是公視發出的新聞或是影片就會有莫名的安全感，很信任他們發出的新聞，他們也有自己旗下員工開設的youtube頻道，裡面的內容不是帶風向，而是教你如何不要被帶風向，是蠻中立客觀的媒體

**Comment by WeiYen1103** [comment]
我覺得媒體識讀的重要性應該在我們小時候就重視，若沒有自己便是問題或是議題的能力，長大後或是出社會很容易失去自己的主見和看法，要自己去求證事實的真實性，而非只聽電視媒體或是報章雜誌的片面之詞。
